spec/models/event_spec.rb

```diff
@@ -101,4 +101,22 @@
       end
     end
   end
+
+  describe "#has_expired?" do
+    context "this event has expired" do
+      let(:event) { create(:event, starts_at: 1.week.ago) }
+
+      it "returns true" do
+        expect(event.has_expired?).to be_truthy
+      end
+    end
+
+    context "this event has not expired" do
+      let(:event) { create(:event, starts_at: 1.week.since) }
+
+      it "returns false" do
+        expect(event.has_expired?).to be_falsey
```

Comment:
If you do use my suggestion to use `subject`, please refactor other similar cases.